### PR TITLE
feat: increase benchmarking table visibility

### PR DESCRIPTION
**Pull request by crfmc**
Corresponds to [Trello Ticket 250](https://trello.com/c/pelxx3O1/250-smaht-embedded-table-scroll-functionality).

* Increase height of benchmarking tables
* Support toggling the benchmarking page descriptions
* Upgrade SPC to v0.1.86

### CODE REVIEW

File: src/encoded/static/components/static-pages/components/benchmarking/BenchmarkingUI.js [hunk 1]
```diff
@@ -18,26 +17,66 @@ export const BenchmarkingLayout = ({
     showBamQCLink = false,
     bamQCHash = '', // TODO: Other datasets will have qcs on same page accessible by anchor hash
     children,
+    setShowInformation = null,
     callout = null,
+    showInformation = true,
```
Comment (Bianca-Morris) [inline src/encoded/static/components/static-pages/components/benchmarking/BenchmarkingUI.js:22]: Is there a reason the useState for these variables is not set in `<BenchmarkingLayout/>`? It's awfully redundant to have to define that in every single data component, and it seems like you're just passing those into this component (the only place where they are actually used), anyway.

Comment (crfmc) [reply src/encoded/static/components/static-pages/components/benchmarking/BenchmarkingUI.js:22]: Sorry yes, definitely redundant. I was previously passing the state down to HashBasedTabController to adjust table height and did not update. Thanks for pointing this out

**Comment by crfmc** [comment]
[SMaHT Embedded Table scroll functionality](https://trello.com/c/pelxx3O1/250-smaht-embedded-table-scroll-functionality)

**Review by Bianca-Morris — APPROVED** [review]

**Comment by crfmc** [comment]
[SMaHT Embedded Table scroll functionality](https://trello.com/c/pelxx3O1/250-smaht-embedded-table-scroll-functionality)

**Pull request merged**